tests/HelperTest.php

```diff
@@ -41,6 +41,35 @@ public function testSet()
         $this->assertSame(['a' => 'A', 'b' => 'B', 'c' => 'C'], $_SESSION);
     }
 
+    public function testAssocSet()
```

Comment:
Realized this test really isn't necessary will remove as it really tests no new functionality